From 7186d0330c3f3e86de577687a82f4ebd96dcb5ac Mon Sep 17 00:00:00 2001
From: Chen Ni <nichen@iscas.ac.cn>
Date: Thu, 26 Feb 2026 11:30:48 +0800
Subject: [PATCH] leds: lgm-sso: Remove duplicate assignments for priv->mmap

Remove duplicate assignment of priv->mmap in intel_sso_led_probe().

Fixes: fba8a6f2263b ("leds: lgm-sso: Fix clock handling")
Signed-off-by: Chen Ni <nichen@iscas.ac.cn>
Link: https://patch.msgid.link/20260226033048.3715915-1-nichen@iscas.ac.cn
Signed-off-by: Lee Jones <lee@kernel.org>
---
 drivers/leds/blink/leds-lgm-sso.c | 2 --
 1 file changed, 2 deletions(-)

--- a/drivers/leds/blink/leds-lgm-sso.c
+++ b/drivers/leds/blink/leds-lgm-sso.c
@@ -809,8 +809,6 @@ static int intel_sso_led_probe(struct pl
 	priv->fpid_clkrate = clk_get_rate(priv->clocks[1].clk);
 
 	priv->mmap = syscon_node_to_regmap(dev->of_node);
-
-	priv->mmap = syscon_node_to_regmap(dev->of_node);
 	if (IS_ERR(priv->mmap)) {
 		dev_err(dev, "Failed to map iomem!\n");
 		return PTR_ERR(priv->mmap);
